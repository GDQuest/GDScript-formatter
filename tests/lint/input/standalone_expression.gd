#region Good

var assigned = "hello"

func good():
	var also_assigned = "world"
	print(assigned)
	print(also_assigned)

	var inner_function = (func():
		var inner_assigned = 42
		print(inner_assigned))

	inner_function.call()

#endregion

#region Bad

"I am not assigned"

func bad():
	"I am not assigned"
	42
	true
	false
	null

#endregion