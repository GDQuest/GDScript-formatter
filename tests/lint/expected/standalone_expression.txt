tests/lint/input/standalone_expression.gd:20:standalone-expression:warning: Standalone expression '"I am not assigned"' is not assigned or used, the line may have no effect
tests/lint/input/standalone_expression.gd:23:standalone-expression:warning: Standalone expression '"I am not assigned"' is not assigned or used, the line may have no effect
tests/lint/input/standalone_expression.gd:24:standalone-expression:warning: Standalone expression '42' is not assigned or used, the line may have no effect
tests/lint/input/standalone_expression.gd:25:standalone-expression:warning: Standalone expression 'true' is not assigned or used, the line may have no effect
tests/lint/input/standalone_expression.gd:26:standalone-expression:warning: Standalone expression 'false' is not assigned or used, the line may have no effect
tests/lint/input/standalone_expression.gd:27:standalone-expression:warning: Standalone expression 'null' is not assigned or used, the line may have no effect